### ISSUE TITLE
`<div data="[object Object]">` in 1.3.0

### ISSUE THREAD

**Issue by pixelastic** [algolia/instantsearch.js#899]
I just moved from 1.2 to 1.3, and I realize that my markup now have a lot of `data="[object Object]"` while inspecting with the Chrome inspector. This has no side effect that I can currently see, but this seems like something that shouldn't be visible in the markup.

## 1.3
![image](https://cloud.githubusercontent.com/assets/283419/13556164/455a9860-e3d3-11e5-9a1a-87375c8e6c5b.png)
![image](https://cloud.githubusercontent.com/assets/283419/13556167/53efbd74-e3d3-11e5-8473-efa083a85470.png)

## 1.2.5
![image](https://cloud.githubusercontent.com/assets/283419/13556177/7b355b78-e3d3-11e5-9776-bb86de0c9edc.png)
![image](https://cloud.githubusercontent.com/assets/283419/13556170/5fbc7eb2-e3d3-11e5-876f-37f16fc7e488.png)

**Comment by vvo** [comment]
Was not able to reproduce this on the dev example, can you try?

**Comment by pixelastic** [comment]
Here is a simpler HTML test case:

```
<!doctype html>
<html>
<head>
</head>
<body>
  <div class="s-hits" id="hits"></div>

  <script src="https://cdn.jsdelivr.net/g/instantsearch.js@1.3.0"></script>
  <script>
    var search = instantsearch({
      appId: 'MXM0JWJNIW',
      apiKey: 'ff7b0847d8705c279222272303e68527',
      indexName: 'steamdb'
    });

    search.addWidget(
      instantsearch.widgets.hits({
        container: '#hits',
        hitsPerPage: 20,
        templates: {
          item: '{{name}}'
        }
      })
    );

    search.start();
  </script>
</body>
</html>
```

![image](https://cloud.githubusercontent.com/assets/283419/13566038/cdbdc594-e453-11e5-994d-2b9a97f52f05.png)

**Comment by vvo** [comment]
will be released as 1.3.2
